README.md

```diff
@@ -26,6 +26,7 @@ Get started on your new app within minutes instead of hours 🚗💨
     - [React](#react)
     - [TypeScript](#typescript)
     - [Hot Module Replacement (HMR)](#hot-module-replacement-hmr)
+    - [JavaScript/TypeScript tests](#javascripttypescript-tests)
```

Comment:
I know we haven't done this everywhere but I'm in favor of capitalizing section headers (Tests)